templates/deployment-zabbix-web.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-web
     app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-web
 spec:
-  replicas: 1
+  replicas: {{ .Values.zabbixweb.replicaCount }}
```

Comment:
Yes, HPA is a good idea. I would leave it for a future release though. I had mostly the matter of high availability in mind (thus the Pod Anti Affinity) so that if a node crashes another pod is still there to take over, and especially with the new Zabbix 6.0 server function this is rather nice to use.
Honestly, I have myself never played with horizontal autoscaling. But it sounds like a great idea, at least for the web frontend. I will play around in a test environment and probably come up with a good solution as a separate PR...?